app/lib/features/member/dialogs/show_change_power_level_dialog.dart

```diff
@@ -140,30 +135,32 @@ class __ChangePowerLevelDialogState extends State<_ChangePowerLevelDialog> {
         ),
         TextButton(
           onPressed: () {
-            if (!_formKey.currentState!.validate()) return;
-            final freshMemberStatus = widget.member.membershipStatusStr();
-            if (freshMemberStatus == currentMemberStatus) {
-              // nothing to do, all the same.
-              Navigator.pop(context, null);
-              return;
-            }
-            int? newValue;
-            if (currentMemberStatus == 'Admin') {
-              newValue = 100;
-            } else if (currentMemberStatus == 'Mod') {
-              newValue = 50;
-            } else if (currentMemberStatus == 'Regular') {
-              newValue = 0;
-            } else {
-              newValue = customValue ?? 0;
-            }
+            if (_formKey.currentState!.validate()) {
+              final freshMemberStatus = widget.member.membershipStatusStr();
+              if (freshMemberStatus == currentMemberStatus) {
+                // nothing to do, all the same.
+                Navigator.pop(context, null);
+                return;
+              }
+              int? newValue;
+              if (currentMemberStatus == 'Admin') {
+                newValue = 100;
+              } else if (currentMemberStatus == 'Mod') {
+                newValue = 50;
+              } else if (currentMemberStatus == 'Regular') {
+                newValue = 0;
+              } else {
+                newValue = customValue ?? 0;
+              }
 
-            if (currentPowerLevel == newValue) {
-              // nothing to be done.
-              newValue = null;
-            }
+              if (currentPowerLevel == newValue) {
+                // nothing to be done.
+                newValue = null;
+              }
 
-            Navigator.pop(context, newValue);
+              Navigator.pop(context, newValue);
+              return;
+            }
```

Comment:
same here ... there was a change made to remove that extra indentation level this PR re-introduces again now ... Probably shouldn't be happening.